storage/api/Storage.Samples/CreateBucketWithTurboReplication.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_create_bucket_turbo_replication]
+
+using System;
+using Google.Apis.Storage.v1.Data;
+using Google.Cloud.Storage.V1;
+
+public class CreateBucketWithTurboReplicationSample
+{
+    public Bucket CreateBucketWithTurboReplication(
+        string projectId = "your-project-id",
+        string bucketName = "your-unique-bucket-name",
+        string location = "your-bucket-location")
+    {
+        var storage = StorageClient.Create();
+        var bucket = storage.CreateBucket(projectId, bucketName);
```

Comment:
I think you can use the CreateBucket overload that takes a Bucket and set all fields from the start, that way you don't have to make to calls (create and update) but just the create one.